openlibrary/catalog/works/find_works.py

```diff
@@ -598,8 +596,6 @@ def update_work_with_best_match(akey, w, work_to_edition, do_updates, fh_log):
         if cur_work['title'] != w['title']:
             print(( 'update work title:', best, repr(cur_work['title']), '->', repr(w['title'])))
         existing_work = get_with_retry(best)
-        if existing_work['type'] != '/type/work':
-            pprint(existing_work)
         assert existing_work['type'] == '/type/work'
```

Comment:
Help the developer to understand failures by adding a string to the end of the assert on line 599
* assert existing_work[‘type’] == ‘/type/work’, “{type} == ‘/type/work’”.format(**existing_work)

Reply:
That is a neat little construct, thanks for the tip